src/box/lua/execute.c

```diff
@@ -346,7 +346,8 @@ lua_sql_bind_decode(struct lua_State *L, struct sql_bind *bind, int idx, int i)
 		 * will be poped from Lua stack.
 		 */
 		buf = xregion_alloc(region, field.sval.len + 1);
-		memcpy(buf, field.sval.data, field.sval.len + 1);
+		memcpy(buf, field.sval.data, field.sval.len);
+		buf[field.sval.len] = '\0';
```

Comment:
I think the `\0` is unnecessary. It's not a C-style string, and its length is described in `bind->bytes`. I think it is better to remove `+1` in the two lines above.

Reply:
Okay, that makes sense. Done.